app/build.gradle

```diff
@@ -4,6 +4,7 @@ apply plugin: 'com.neenbedankt.android-apt'
 apply plugin: 'hu.supercluster.paperwork'
 apply from: '../code_quality_tools/jacoco.gradle'
 apply plugin: 'com.frogermcs.androiddevmetrics'
+apply from: '../code_quality_tools/quality.gradle'
```

Comment:
@artem-zinnatullin Findbugs for a Java project won't work since this path `/build/intermediates/classes` does not exist.

Also I think there are some other things that are a bit different in Java.